app/lib/annual_report/top_hashtags.rb

```diff
@@ -17,6 +17,12 @@ def generate
   private
 
   def top_hashtags
-    Tag.joins(:statuses).where(statuses: { id: report_statuses.select(:id) }).group(:id).having('count(*) > 1').order(total: :desc).limit(SET_SIZE).pluck(Arel.sql('COALESCE(tags.display_name, tags.name), count(*) AS total'))
+    Tag.joins(:statuses).where(statuses: { id: report_statuses.select(:id) }).group(coalesced_tag_names).having('count(*) > 1').order(count_all: :desc).limit(SET_SIZE).count
```

Comment:
I would have expected grouping on the coalesced tag names to be a significant difference, but it appears not to be. The query plan is almost identical and the cost is identical. Actual run time is the same as well.

Reply:
I thought so too -- given indexed column there, but saw same thing so included it.